modules/handlebarsUtils.js

```diff
@@ -126,22 +151,31 @@
     });
```

Comment:
```
dotsRemoved = camelCaseRemoved.replace(/\./g, function() {
   return '_';
}
```

Is this what makes the string value use "_" instead of "-"?